test/e2e/multihoming.go

```diff
@@ -674,6 +674,56 @@ var _ = Describe("Multi Homing", feature.MultiHoming, func() {
 				true, // scheduled on the same node
 				Label("BUG", "OCPBUGS-59657"),
 			),
+			Entry(
+				"can be reached by a client pod in the host network on a different node, when the localnet uses a VLAN and an external router",
+				networkAttachmentConfigParams{
+					name:     secondaryNetworkName,
+					topology: "localnet",
+					vlanID:   localnetVLANID,
+				},
+				podConfiguration{ // client on host network
+					name:         clientPodName,
+					hostNetwork:  true,
+					isPrivileged: true,
+				},
+				podConfiguration{ // server attached to localnet secondary network
+					attachments: []nadapi.NetworkSelectionElement{{
+						Name: secondaryNetworkName,
+					}},
+					name:                podName,
+					containerCmd:        httpServerContainerCmd(port),
+					ipRequestFromSubnet: fromExternalNetwork,
+					isPrivileged:        true,
+					usesExternalRouter:  true,
+				},
+				false, // scheduled on distinct Nodes
+				Label("BUG", "OCPBUGS-59657"),
+			),
+			Entry(
+				"can be reached by a client pod in the host network on the same node, when the localnet uses a VLAN and an external router",
+				networkAttachmentConfigParams{
+					name:     secondaryNetworkName,
+					topology: "localnet",
+					vlanID:   localnetVLANID,
+				},
+				podConfiguration{ // client on host network
+					name:         clientPodName,
+					hostNetwork:  true,
+					isPrivileged: true,
+				},
+				podConfiguration{ // server attached to localnet secondary network
+					attachments: []nadapi.NetworkSelectionElement{{
+						Name: secondaryNetworkName,
+					}},
+					name:                podName,
+					containerCmd:        httpServerContainerCmd(port),
+					ipRequestFromSubnet: fromExternalNetwork,
+					isPrivileged:        true,
+					usesExternalRouter:  true,
+				},
+				true, // scheduled on the same node
+				Label("BUG", "OCPBUGS-59657"),
```

Comment:
what are these Label's used for? these are for known issues right? but since we have fixed this issue why are downstream bugs being linked? we should remove these